### DISCUSSION TITLE
dns validation specify different domain

### DISCUSSION THREAD

**Discussion by speedygonzales391** [go-acme/lego#1812] [Q&A] [answered]
Is it posible add new command line option for usage to specify different domain for cname validation? For example acme.sh has this feature https://github.com/acmesh-official/acme.sh/wiki/DNS-alias-mode

**Comment by ldez** [comment, marked as answer]
Hello,

lego is already able to do that, by default, you just have to create a CNAME.

https://letsencrypt.org/2019/10/09/onboarding-your-customers-with-lets-encrypt-and-acme.html#the-advantages-of-a-cname
